src/main/webapp/index.html

```diff
@@ -5,8 +5,13 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://apis.google.com/js/platform.js" async defer></script>
+  <meta name="google-signin-client_id" content="757099697912-i6jll98mfgochdo2vgjcovf64pepjesc.apps.googleusercontent.com">
+
+
   <script type="module" src="scripts/index-script.js" defer></script>
-  <link rel="stylesheet" href="styles/general.css">
```

Comment:
Is this not needed any more for consistent styling?

Reply:
I honestly dont remember deleting that, but i just ran it on a server and all the styling seems to be fine?